components/projects/NewProject/NewCustomer.tsx

```diff
@@ -37,6 +38,7 @@ function NewCustomer(): JSX.Element {
     });
     const url = `${process.env.NEXT_PUBLIC_API_URL}/customer`;
     const { mutate } = useSWRConfig();
+    const idJwt = useUser()?.getSignInUserSession()?.getIdToken().getJwtToken();
```

Comment:
Now you can remove the token definitions and useUser imports from the components